en/lesson-12.md

```diff
@@ -188,6 +188,53 @@ In `biblatex`, the reference style is picked when we load the package. Here,
 we've used `authoryear`, but there is a `numeric` style and many others are
 also available.
 
+## Choosing between the BibTeX workflow and `biblatex`
+
+Even though both the BibTeX workflow and `biblatex` get their input via BibTeX
+files and can produce structurally similar output in the document, they use
+completely different ways to produce this result. That means that there are
+some differences between the two approaches that may help you choose which
+one works best for you.
+
+In the BibTeX workflow the bibliography style is ultimately decided
+by a `.bst` file, whose name you give with `\bibliographystyle` command.
+`biblatex` does not use `.bst` files and uses a different system.
+If you are using a template that comes with a `.bst` file or are given a `.bst`
+file for your project, you must use the BibTeX workflow and cannot use
+`biblatex`.
+
+The different approach `biblatex` takes implies that you can modify the output
+of the bibliography and citation commands directly from your document preamble
+using LaTeX-based commands. Modifications of BibTeX `.bst` styles on the other
+hand usually require working with these external files and need knowledge of
+the BibTeX programming language. Generally speaking, `biblatex` is said to be
+easier to customise than the BibTeX workflow.
+
+In `biblatex` it is generally easier to implement more elaborate citation
+styles with a wider array of different citation commands. It also offers more
+context-dependent features. Roughly speaking this is less interesting for
+the styles common in many STEM subjects, but becomes relevant for some more
+complex styles in some areas of the humanities.
+
+BibTeX can only sort US-ASCII characters correctly and relies on workarounds
+to provide US-ASCII-based sorting for non-US-ASCII characters.
+With Biber `biblatex` offers full Unicode sorting capabilities. Thus `biblatex`
```

Comment:
`With Biber` -> `With Biber,` (not sure)

Reply:
I know nothing about punctuation in English and while I like to set off some 'introductory' phrases with a comma, that doesn't quite feel right for 'With Biber' especially since the sentence is so short. I didn't include this change in #78.

Reply:
@moewew No problem.  As I said I'm not sure: my English is not that good.  I just suggested the comma there because it sounds to me like two parts of a sentence a little “too close together”.  But no problem, really :-)

Reply:
@PhelypeOleinik I was hoping Barbara would go through the texts again at some point and that she can illuminate us about that comma. (And pick up on the other typos and unfortunate formulations that are no doubt elsewhere in the text.)